test/gtest/common/test_helpers.h

```diff
@@ -517,19 +517,25 @@ class ptr_vector_base {
     typedef std::vector<T*> vec_type;
     typedef typename vec_type::const_iterator const_iterator;
 
-    ptr_vector_base() {
-    }
+    ptr_vector_base() : m_init_by_list(false) { }
+
+    constexpr ptr_vector_base(std::initializer_list<T*> init_list) :
+            m_vec(init_list), m_init_by_list(true) { }
 
     virtual ~ptr_vector_base() {
-        clear();
+        if (!m_init_by_list) {
```

Comment:
why needed?

Reply:
> why needed?

it's needed to not release entities came inside the initializer list

Reply:
it looks weird... maybe can have some better solution?

Reply:
> it looks weird... maybe can have some better solution?

yes, found another one which is more suitable.
overriding conversation from `ucs::ptr_vector` to `std::vector` helps